src/main/kotlin/nomic/api/RulesAmendments.kt

```diff
@@ -0,0 +1,50 @@
+package nomic.api
+
+import nomic.data.EntityNotFoundException
+import nomic.domain.entities.RulesAmendmentsModel
+import nomic.domain.rulesamendments.RuleAmendmentDomain
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.ExceptionHandler
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("api/rules_amendments")
+class RulesAmendments(val ruleAmendmentDomain: RuleAmendmentDomain) {
+    // Path to endpoint is api/rules_amendments/ExistingGameId
+    @GetMapping("{gameid}", produces = ["application/json;charset=UTF-8"])
+    fun getRulesAmendments(@PathVariable(value = "gameid") gameId: String): ResponseEntity<Any> {
+        val rulesAmendments: MutableList<RulesAmendmentsModel> = ruleAmendmentDomain.getRulesAmendments(gameId)
```

Comment:
I'll grant you that this is wading chest high into a huge debate in the software dev community, but do you need to still explicitly list the type? Do you think it could be clear enough to infer from the other side? Granted, the other side says nothing about mutability.